cmd/cephcsi.go

```diff
@@ -90,6 +90,7 @@ func init() {
 		"skip image flattening if kernel support mapping of rbd images which has the deep-flatten feature")
 
 	flag.BoolVar(&conf.Version, "version", false, "Print cephcsi version information")
+	flag.BoolVar(&conf.EnableProfiling, "enableprofiling", false, "enable go profiling")
```

Comment:
helm template update is missing for this one.

Reply:
yes, working on helm charts. Will update the PR once that is done. and Also, will be adding documentation once done.